keyform/templates/keyform/home.html

```diff
@@ -33,12 +33,12 @@ <h2 class="title">{% trans 'Keyform Requests' %}</h2>
                     </div>
                     <div class="small-2 large-2 columns text-center">
                         {% for key in object.keydata_set.all %}
-                            {{ key.room_number }}{% if not forloop.last %},{% endif %}&nbsp;
+                            {% if key.room_number != '' %}{{ key.room_number }}{% if not forloop.last %},{% endif %}&nbsp;{% endif %}
                         {% endfor %}
                     </div>
                     <div class="small-2 large-2 columns text-center">
                         {% for key in object.keydata_set.all %}
-                            {{ key.core_number }}{% if not forloop.last %},{% endif %}&nbsp;
+                            {% if key.core_number != '' %}{{ key.core_number }}{% if not forloop.last %},{% endif %}&nbsp;{% endif %}
```

Comment:
You should use indentation and whitespace here instead of putting everything on the same line. You could also use falsy evaluation here instead of explicitly comparing to the empty string if you feel like it.